config/main.py

```diff
@@ -138,7 +138,7 @@ def _validate_interface_mode(ctx, breakout_cfg_file, interface_name, target_brko
         return False
 
     # Check whether the  user-selected interface is part of  'port' table in config db.
-    if interface_name not in port_dict.keys():
+    if interface_name not in list(port_dict.keys()):
```

Comment:
>list [](start = 29, length = 4)

The same

Reply:
This change was made by the `2to3` tool (see [here](https://www.python.org/dev/peps/pep-0469/#migrating-directly-to-python-3)):

> Note that in many cases d.keys() can be replaced by just d, but the 2to3 migration tool doesn't attempt that replacement.

Plus, keeping `.keys()` makes it clear what the intention is.